x-pack/test/api_integration/apis/management/index_management/indices.ts

```diff
@@ -119,13 +106,15 @@ export default function ({ getService }) {
         const {
           body: { indices: indices1 },
         } = await indexStats(index, 'flush');
+        // @ts-ignore
```

Comment:
why is this method not returning with the right type? 🤔

Reply:
I might not need this actually, I think the return type is correct, I'll check